stylix/maintainers.nix

```diff
@@ -29,4 +29,10 @@
     github = "skoove";
     githubId = 53106860;
   };
+  themaxmur = {
+    email = "muravjev.mak@yandex.ru";
+    name = "Maxim Muravev";
+    github = "TheMaxMur";
+    githubId = 31189199;
+  };
```

Comment:
Are you not already in the Nixpkgs list since commit https://github.com/NixOS/nixpkgs/commit/c855b309b5ee195547cdb8aab123c00b9cb483c5:

```suggestion
```